snapcraft/plugins/kernel.py

```diff
@@ -173,12 +173,18 @@ def _set_kernel_targets(self):
                 self.options.kernel_image_target[self.project.deb_arch]
 
         self.make_targets = [self.kernel_image_target, 'modules']
+        self.make_install_targets = [
+            'modules_install', 'INSTALL_MOD_PATH={}'.format(self.installdir)]
         self.dtbs = ['{}.dtb'.format(i)
                      for i in self.options.kernel_device_trees]
         if self.dtbs:
             self.make_targets.extend(self.dtbs)
-        self.make_install_targets = [
-            'modules_install', 'INSTALL_MOD_PATH={}'.format(self.installdir)]
+        elif (self.project.kernel_arch == "arm" or
+                self.project.kernel_arch == "arm64"):
+            self.make_targets.append('dtbs')
+            self.make_install_targets.extend([
+                'dtbs_install',
+                'INSTALL_DTBS_PATH={}/dtbs'.format(self.installdir)])
```

Comment:
We also run this tests in armhf and arm64. The tests were not updated to take into account that they could be running on these architectures, so now we have new failures:

https://objectstorage.prodstack4-5.canonical.com/v1/AUTH_77e2ada1e7a84929a74ba3b87153c0ac/autopkgtest-xenial-snappy-dev-snapcraft-daily/xenial/armhf/s/snapcraft/20170307_030329_f94e4@/log.gz

This is pretty important, because we can't land in the archives if the tests start failing. @piso77, can you please take a look?